test/e2e/antreapolicy_test.go

```diff
@@ -5368,26 +5380,34 @@ func TestFQDNCacheMinTTL(t *testing.T) {
 	require.NoError(t, data.setPodAnnotation(data.testNamespace, "custom-dns-server", "test.antrea.io/random-value",
```

Comment:
not introduced in this PR, but I just noticed the following assertion above:
```
	assert.EventuallyWithT(t, func(t *assert.CollectT) {
		_, err := curlFQDN(testFQDN)
		assert.NoError(t, err)
	}, 2*time.Second, 1*time.Millisecond, "failed to curl test FQDN: ", testFQDN)
```

1ms is much too frequent to be executing the condition function. Please update to 100ms.